I see the CI build failed, rightly due to:  

/home/travis/build/FaradayRF/Faraday-Software/Applications/Telemetry/telemetry.py:25:1: F401 'flask_cors.cross_origin' imported but unused

To avoid this in future, what do I need to do to enable strict checking or to run the CI check itself before I submit?

--- a/Applications/Telemetry/telemetry.py
+++ b/Applications/Telemetry/telemetry.py
@@ -22,7 +22,7 @@ import json
 
 from flask import Flask
 from flask import request
-from flask_cors import CORS, cross_origin
+from flask_cors import CORS
 
 # Can we clean this up?
 sys.path.append(os.path.join(os.path.dirname(__file__), "../../Faraday_Proxy_Tools/"))  #Append path to common tutorial FaradayIO module

